docs/src/pages/premium-themes/PremiumThemes.js

```diff
@@ -56,6 +56,18 @@ const themes = [
     category: 'Components',
     href: 'https://www.creative-tim.com/product/material-kit-react?partner=104080',
   },
+  {
+    name: 'Firebase-UI',
```

Comment:
I'd suggest we sear clear of the Firebase trademark. How about `Appbase`? 🙂 

I haven't flagged all the other references to Firebase.

Reply:
`Paperbase`